app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  # Signs In the user
+  def sign_in(user)
+    session[:user_id] = user.id
+  end
+
+  # Returns the current signed in User (if any)
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+end
```

Comment:
Agreed - helpers will be removed.

Reply:
There is no /api/v1/ scope for ApplicationController at the moment.
current_user could reside in SessionsController for now.
Could also be in UsersController.

Reply:
current_user method should work if it is moved to application_controller (just tried it locally) as a public method.
The reason we need it in application_controller is because current_user needs to be accessible throughout the application.